Changed relationships implementation (#269)
desmos-labs/desmos: docs/developers/msgs/create-relationship.md

  "type": "desmos/MsgCreateRelationship",
  "value": {
    "sender": "<Desmos address that's creating the relationship>",
    "receiver": "<Desmos address that's receiving the relationship>",

>     "subspace": "<Subspace of a relationship>"

Reviewer: ```suggestion
    "subspace": "<Subspace of the relationship>"
```

| :-------: | :----: | :-------- |
| `sender`  | String | Desmos address of the user that is creating the relationship |
| `receiver`| String | Desmos address of the relationship's recipient |

> | `subspace`| String | Required string that identifies the app where the user make the relationship |

Reviewer: ```suggestion
| `subspace`| String | Identifies the app for which the the relationship should be valid |
```
